grails-app/views/stockMovement/_documents.gsp

```diff
@@ -52,9 +52,27 @@
                                         <warehouse:message code="default.button.delete.label"/>
                                     </g:link>
                                 </g:if>
-                                <g:link url="${document.uri}" target="_blank" class="button">
-                                    <warehouse:message code="default.button.download.label"/>
-                                </g:link>
+                                <g:if test="${document.downloadOptions}">
+                                    <span class="action-menu">
+                                        <button class="action-btn button">
+                                            <warehouse:message code="default.button.download.label"/>
```

Comment:
I know that you are not the person who added this line, but we are using g:message
```suggestion
                                            <g:message code="default.button.download.label"/>
```

Reply:
I'm going to leave it as is since warehouse:message is used everywhere in the file and I want to refactor as little as possible right now, but that's good to know